lib/analysis_options.yaml

```diff
@@ -0,0 +1,2 @@
+analyzer:
```

Comment:
Although this works, there shouldn't be an `analysis_options.yaml` in the `lib` folder, but not sure why the main `analysis_options.yaml` isn't catching the `classes.g.dart` file

Reply:
@maxxfrazer @tadaspetra It's a bug of `pana`, the better way is to add a `build.yaml` file to add the `// ignore_for_file` for the `*.g.dart` files.
```yaml
targets:
  $default:
    builders:
      source_gen|combining_builder:
        options:
          ignore_for_file:
          - non_constant_identifier_names
          - deprecated_member_use_from_same_package
          - unused_element
```

see: https://github.com/dart-lang/pana/issues/1013#issuecomment-1034703342